src/main/java/stsjorbsmod/actions/IncreaseManifestAction.java

```diff
@@ -34,14 +32,18 @@ public void update() {
         Cull player = (Cull) target;
 
         if (this.duration == this.startDuration) {
-            String msg = String.format(TEXT[0], amount);
+
+            String msg = amount < 0 ?  String.format(TEXT[1], amount) : String.format(TEXT[0], amount);
             AbstractDungeon.effectsQueue.add(new TextAboveCreatureEffect(target.hb.cX - target.animX, target.hb.cY, msg, Settings.RED_TEXT_COLOR));
```

Comment:
Should probably also make the color of the effect green if the amount is negative